charts/config-validator/values.yaml

```diff
@@ -35,23 +35,36 @@ networkPolicy:
 # on which nodes to run the Forseti on-GKE pods.
 nodeSelectors: []
 
-gitSync:
-  # gitSync.image is the container image used by the config-validator git-sync side-car
-  image: gcr.io/google-containers/git-sync
-
-  # gitSync.imageTag is the container image tag used by the config-validator git-sync side-car
-  imageTag: v3.1.2
+policyLibrary:
+  # bucket is the GCS storage bucket containing the policy-library.  This overrides policyLibrary.respositoryURL. Ommit the gs://.
+  bucket: ""
```

Comment:
Could bucket and bucketFolder be combined into one value?

Reply:
@gkowalski-google In principle that makes sense as it would reduce the number of variables.  However, the only information we get from TF is the name of the bucket.  So we are left with concatenating the name of the bucket and path anyway.  In the TF module, the path is hard coded.

I am interested in your thoughts here.
1. Do we hard code the path?  This reduces the number of variables.  This also maintains parity with the TF module .
2. Parameterize the path in Helm, making the defaults as they are found in the TF module

Reply:
I think the right approach is to have TF pass the full path into Helm. Currently TF does not expose this anywhere, but perhaps it should? Consider adding a variable to the TF vars for the Policy Library GCS folder. Then TF can concatenate the bucket + folder as a single value to Helm. This would require changes outside the scope of this work, so an option is to address this separately in another issue.

Reply:
Might I recommend that we proceed as is in this PR but open another issue like you suggested?

1. In the terraform-google-forseti repo
2. In the helm charts repo to change this parttern in all the charts.

Reply:
Agreed. Please consider any changes to your PR to ensure this future change will be backwards compatible. For example, update Helm to only accept a single value which either includes the `policy-library` text or hard-codes it at the end. I've created this issue for now, maybe we can avoid a future Helm change:
- https://github.com/forseti-security/terraform-google-forseti/issues/358